set.mm

```diff
@@ -98936,6 +99138,16 @@ is that it denies the existence of a set containing itself ( ~ elirrv ).
     ( cvv wcel wa wn cep wfr zfregfr efrn2lp mpan elex anim12i con3i pm2.61i )
     ACDZBCDZEZABDZBADZEZFZCGHRUBCICABJKUARSPTQABLBALMNO $.
 
+  $( A class cannot be an element of one of its elements.  (Contributed by AV,
+     14-Jun-2022.) $)
+  elnotel $p |- ( A e. B -> -. B e. A ) $=
+    ( wcel en2lp imnani ) ABCBACABDE $.
+
+  $( A class cannot be an element of one of its elements.  (Contributed by AV,
+     14-Jun-2022.) $)
+  elnel $p |- ( A e. B -> B e/ A ) $=
```

Comment:
necessary ? elnotel probably suffices

Reply:
This is the old philosophical discussion: ~elnotel is useful for proofs, but ~elnel has a better looking representation.